docx/header.py

```diff
@@ -8,12 +8,11 @@
     absolute_import, division, print_function, unicode_literals
 )
 
-from .shared import ElementProxy
+from .blkcntnr import BlockItemContainer
 
 
-class Header(ElementProxy):
+class Header(BlockItemContainer):
     """
-    The default page header of a section.
+    Proxy for ``<w:hdr>`` element in this section, having primarily a
+    container role.
```

Comment:
I'm not sure yet we can directly inherit from `BlockItemContainer`. Let's leave it as `object` for now. I suspect we're going to need something like `Header.body` where the body is actually the block item container. The reason is this object has to intercept calls that need it to add the header part automatically and so forth. Anyway, we don't need block item functionality just yet, so let's just push that inheritance decision off until we get to needing it :)